### ISSUE TITLE
Any workaround for brave browser?

### ISSUE THREAD

**Issue by RameshKumarGogoi** [ActivityWatch/aw-watcher-web#91]
Raw data exist but it shows "No Data" in the browser tab.

**Comment by felladrin** [comment]
Yeah, using the extension in Brave does log usage, but the Browser Activity tab remains empty.

<img width="1182" alt="image" src="https://user-images.githubusercontent.com/418083/195182743-3eaefe77-cffa-486c-a0e5-b4b5279d0975.png">

<img width="588" alt="image" src="https://user-images.githubusercontent.com/418083/195182577-a32bd8e1-8132-46c1-8c36-46e6d42638cd.png">

<img width="415" alt="image" src="https://user-images.githubusercontent.com/418083/195182957-a5febcd5-dec0-485f-a09b-37befe7af31f.png">

**Comment by shan100github** [comment]
It looks like it's still  Browser Activity tab remains empty in the Brave browser, looking forward solution if any.

**Comment by sanderlienaerts** [comment]
Temporary fix (MacOS): https://github.com/ActivityWatch/activitywatch/issues/461#issuecomment-1269244862 
Fix depends on https://github.com/ActivityWatch/aw-webui/pull/283
